src/js/components/ExpandableGroup/ExpandableGroup.tsx

```diff
@@ -17,6 +17,7 @@ const propTypes = {
   onLabelBlur: PropTypes.func,
   children: PropTypes.node.isRequired,
   mods: PropTypes.string,
+  closedHelperText: PropTypes.string || PropTypes.node,
```

Comment:
It might be a better name to call this `groupHelperText` because if design later decides we want this text to display when the panel is open `closedHelperText` doesn't make sense there